compose/src/main/java/com/bottlerocketstudios/compose/widgets/UserCard.kt

```diff
@@ -0,0 +1,14 @@
+package com.bottlerocketstudios.compose.widgets
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
```

Comment:
Either document this liberally with TODOs for its intended purpose, or remove completely.

Reply:
Should have been removed; went with a different idea.